src/rocksdb_options.rs

```diff
@@ -2081,11 +2081,11 @@ impl IngestExternalFileOptions {
             );
         }
     }
-    
-    pub fn get_write_global_seqno(&self) -> bool {
-        unsafe { 
-	    crocksdb_ffi::crocksdb_ingestexternalfileoptions_get_write_global_seqno(self.inner)
-	}
+
+    pub fn get_write_global_seqno(&mut self) -> bool {
+        unsafe {
+            crocksdb_ffi::crocksdb_ingestexternalfileoptions_get_write_global_seqno(self.inner)
+        }
```

Comment:
rollback?

Reply:
My fault, I did not noticed `mut` here, I thought there was only a format problem. I will restore changes here.